cmd/dcrdata/public/js/helpers/http.js

```diff
@@ -0,0 +1,26 @@
+
+export async function requestJSON (url, reqBody) {
```

Comment:
Noticing that we're never actually using the `reqBody` arg. Should we simplify?

Reply:
Ah right, there's no proper POST version of this.  Might as well simplify